jwst/datamodels/schema_editor.py

```diff
@@ -480,47 +470,60 @@ class Options:
+         keyword database and the output directory you want the changed model
+         schemas written to. Then it will ask you what kind of changes you wish
+         to make to the schema files. (Additions to the schema, deletions, and
+         so on.) Then it will determine the differences, display them one at a
+         time, and ask if you want to make the change. If you say yes, the
+         change will be made to the schema. Finally, it will create a new
+         subdirectory in the output directory whose name starts with "schemas"
+         and write all modified schemas to that directory.
+
+         If schema are modified, and keywords are omitted, a file
+         `aaa_omitted_keywords.yaml` will be created. This file can be used as
+         the input to the `--omit_file` option
+
+         It will also write the options to a file in your home directory so
+         that the next time the script is run it will not ask you about the
+         same changes twice. The name of this file is
+         `schema_editor_options.txt`. It is a text file, you can edit it or
+         delete it. You are seeing the message because no options file was
+         found in your home directory. If one is found, this message will not
+         be displayed.
 
          The first two inputs are the names of the input and output directories.
          If you leave them blank, this script will use the current directory.
          All the following input to the script should either be y (yes) or n (no).
          Each question has a default answer which will be displayed as a
          capital letter (Y or N). If you hit return, the script will use
          the default value.
+
         """
 
     run_script = "do you want to continue running the editor"
 
+    # Command line options. 2 or 3-tuple where:
+    #    - name: Parameter name and full option name
+    #    - help: The help text
+    #    - Create a short option. If not present or True, a single character option,
+    #      based on the first character of the name, is created.
     prompts = (
-               ("input", "directory name containing keyword database"),
-               ("output", "directory name model schemas will be written to"),
-               ("add", "fields in the keyword db but not in the model to the model") ,
-               ("delete", "fields in the model but not in the keyword db from the model") ,
-               ("edit", "fields found in both to match the values in the keyword db") ,
-               ("rename", "fields in the model to match the names in the keyword db") ,
-               ("list", "changes without making them"),
-               ("query", "for approval of changes before they are made"),
-               ("omit", "")
-            )
+        ("input", "directory name containing keyword database"),
+        ("output", "directory name model schemas will be written to"),
+        ("add", "fields in the keyword db but not in the model to the model"),
```

Comment:
Actually, this is technically right. These were apparently setup to read as one whole string, so that sentence is

"add fields in the keyword db but not in the model to the model"

This is how it appears when doing the "-h" or when that option is shown in the interactive mode.  I'll update to

"add fields in the keyword db but not in the model to the model schema"

Reply:
Perhaps they would be more readable/intelligible if the object of the action immediately followed the action, e.g "add fields to the model schema that are in the keyword db but not the model", etc.